CMakeLists.txt

```diff
@@ -638,12 +638,8 @@ endif()
 
 if(UNIX)
   if(APPLE)
-    # TODO: temp for #5281 - Resigned with a different entitlements to avoid an issue when we pip install stuff into the E+ dir with native comps (numpy for eg)
-    set(ENERGYPLUS_REPO "jmarrec")
-    set(ENERGYPLUS_RELEASE_NAME "v24.2.0a-entitlements")
-
     if (ARCH MATCHES "arm64")
-      set(ENERGYPLUS_EXPECTED_HASH 29616d6aa23e2fc0f71362da9794ef08)
+      set(ENERGYPLUS_EXPECTED_HASH 8a7336112955e90c525cf78fd33ce927)
```

Comment:
Unfortunately that release is missing the intel mac package, and doesn't have the arm64 ubuntu ones.

We could use my release here: https://github.com/jmarrec/EnergyPlus/releases/tag/v25.1.0-IOFreeze-Ubuntu-arm64-macfixup